tests/flow/test_effects.py

```
@@ -0,0 +1,855 @@
+    def test16_rerun_disable_effects(self):
+        # test replication works when effects are disabled
+
+        # no leftovers from previous test
+        self.env.assertFalse(self.monitor_containt_effect())
+
+        # update graph key
+        global GRAPH_ID
+        GRAPH_ID = "effects_disabled"
+
+        # update graph objects to use new graph key
+        self.master_graph = Graph(self.master, GRAPH_ID)
```

Ruff (0.12.2)

`Graph` may be undefined, or defined from star imports

(F405)